src/components/Book.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Book = ({ title, author }) => (
+  <>
+    <h3>{title}</h3>
+    <h4>{author}</h4>
```

Comment:
- [ ] Nice working on your Book Component, as a requirement kindly add a `button`  for removing books.